frontend/test/pytest/test_custom_devices.py

```diff
@@ -22,6 +22,73 @@
 from catalyst.compiler import get_lib_path
 from catalyst.utils.exceptions import CompileError
 
+# These have to match the ones in the configuration file.
+OPERATIONS = [
+    "QubitUnitary",
+    "PauliX",
+    "PauliY",
+    "PauliZ",
+    "MultiRZ",
+    "Hadamard",
+    "S",
+    "T",
+    "CNOT",
+    "SWAP",
+    "CSWAP",
+    "Toffoli",
+    "CY",
+    "CZ",
+    "PhaseShift",
+    "ControlledPhaseShift",
+    "RX",
+    "RY",
+    "RZ",
+    "Rot",
+    "CRX",
+    "CRY",
+    "CRZ",
+    "CRot",
+    "Identity",
+    "IsingXX",
+    "IsingYY",
+    "IsingZZ",
+    "IsingXY",
+    "SX",
+    "ISWAP",
+    "PSWAP",
+    "SISWAP",
+    "SQISW",
+    "CPhase",
+    "BasisState",
+    "QubitStateVector",
+    "StatePrep",
+    "ControlledQubitUnitary",
+    "DiagonalQubitUnitary",
+    "SingleExcitation",
+    "SingleExcitationPlus",
+    "SingleExcitationMinus",
+    "DoubleExcitation",
+    "DoubleExcitationPlus",
+    "DoubleExcitationMinus",
+    "QubitCarry",
+    "QubitSum",
+    "OrbitalRotation",
+    "QFT",
+    "ECR",
+    "Adjoint(S)",
+    "Adjoint(T)",
+    "Adjoint(SX)",
+    "Adjoint(ISWAP)",
+    "Adjoint(SISWAP)",
+    "MultiControlledX",
+]
+OBSERVABLES = [
```

Comment:
Do you want to update these obs names here?

Reply:
Thanks! https://github.com/PennyLaneAI/catalyst/pull/369/commits/35662bd46c7ffb9eacd232d6a8515f7d4cfeecac